src/CssInliner.php

```diff
@@ -190,17 +179,10 @@ public function inlineCss(string $css = ''): self
         if ($this->isStyleBlocksParsingEnabled) {
             $combinedCss .= $this->getCssFromAllStyleNodes();
         }
-
-        $cssWithoutComments = $this->removeCssComments($combinedCss);
-        [$cssWithoutCommentsCharsetOrImport, $cssImportRules]
-            = $this->extractImportAndCharsetRules($cssWithoutComments);
-        [$cssWithoutCommentsOrUninlinableAtRules, $cssAtRules]
-            = $this->extractUninlinableCssAtRules($cssWithoutCommentsCharsetOrImport);
-
-        $uninlinableCss = $cssImportRules . $cssAtRules;
+        $cssDocument = new CssDocument($combinedCss);
```

Comment:
I suggest the name `$parsedCss`.

Reply:
Renamed the variable (and likewise the parameter of the methods it's passed to).